tests/suite/test_virtual_server_grpc.py

```diff
@@ -134,6 +134,10 @@ def test_connect_grpc_backend(self, kube_apis, ingress_controller_prerequisites,
             except grpc.RpcError as e:
                 print(e.details())
                 pytest.fail("RPC error was not expected during call, exiting...")
+        # Assert grpc_status is in the logs. The gRPC response in a successful call is 0.
+        ic_pod_name = get_first_pod_name(kube_apis.v1, ingress_controller_prerequisites.namespace)
+        last_log = get_last_log_entry(kube_apis.v1, ic_pod_name, ingress_controller_prerequisites.namespace)
+        assert '"POST /helloworld.Greeter/SayHello HTTP/2.0" 200 0' in last_log
```

Comment:
the status for this case is taken from the trailer ($upstream_trailer_grpc_status). does it make sense to also check the status for the case when the status is taken from the header ($sent_http_grpc_status) ?